app/views/colors/index.html.erb

```diff
@@ -1,8 +1,59 @@
-<div class="trafficlight">
-  <div class="protector"></div>
-  <div class="protector"></div>
-  <div class="protector"></div>
-  <div class="red<%= " lighted" if @colors[:red] %>"></div>
-  <div class="yellow<%= " lighted" if @colors[:yellow] %>"></div>
-  <div class="green<%= " lighted" if @colors[:green] %>"></div>
-</div>
\ No newline at end of file
+<div class="messages">
+  <% if @colors[:red] %>
```

Comment:
@danielmorrison These three color-based conditional wrapping tags should only be here temporarily. I'd love to write a script that runs every 5s or so to keep the webpage up to date. At that point, we'll want to keep these `<div class="message">` elements around at all times so that we can hide/show them (after updating their contents via JS) as the status changes.